nmigen_soc/csr/event.py

```diff
@@ -0,0 +1,106 @@
+# nmigen: UnusedElaboratable=no
```

Comment:
Disabling the warning in `test_csr_event.py` is not enough, because `self._mux` elaboratable defined in `.__init__()` would also be unused. So I disabled it here too.